src/bin/test_mpc.rs

```diff
@@ -109,27 +110,41 @@ async fn main() -> Result<(), Box<dyn Error>> {
             }
             InputType::Int64 => panic!("Only field values are supported"),
         },
-        TestAction::SemiHonestIPA => match args.input.input_type {
-            InputType::Fp31 => {
-                let query_config = QueryConfig {
-                    field_type: FieldType::Fp31,
-                    query_type: QueryType::Ipa(IpaQueryConfig {
-                        per_user_credit_cap: 3,
-                        max_breakdown_key: 3,
-                        num_multi_bits: 3,
-                        attribution_window_seconds: 0,
-                    }),
-                };
-                let query_id = clients[0].create_query(query_config).await.unwrap();
-                let output =
-                    semi_honest::<Fp31, MatchKey, BreakdownKey>(input, &clients, query_id).await;
-                print_output(&output);
-            }
-            InputType::Fp32BitPrime => {
-                unimplemented!()
+        TestAction::SemiHonestIPA => {
```

Comment:
See #622, which also includes this change.  Though this is a nicer factoring of the other pieces.